tests/FakeItEasy.Specs/ConfiguringPropertySetterSpecs.cs

```diff
@@ -110,6 +112,7 @@ public interface IHaveInterestingProperties
                 .x(() => exception.Message.Should().Be("The specified expression is not a method call or property getter."));
         }
 
+        [SuppressMessage("Microsoft.Globalization", "CA1305:SpecifyIFormatProvider", MessageId = "System.String.Format(System.String,System.Object)", Justification = "Not relevant in this case")]

```

Comment:
I'm wondering if we shouldn't switch off CA1305 for the test projects?
On 19 May 2016 22:34, "Thomas Levesque" <notifications@github.com> wrote:

> In tests/FakeItEasy.Specs/ConfiguringPropertySetterSpecs.cs
> <https://github.com/FakeItEasy/FakeItEasy/pull/713#discussion_r63947325>:
>
> > @@ -110,6 +112,7 @@ public interface IHaveInterestingProperties
> >                  .x(() => exception.Message.Should().Be("The specified expression is not a method call or property getter."));
> >          }
> >
> > +        [SuppressMessage("Microsoft.Globalization", "CA1305:SpecifyIFormatProvider", MessageId = "System.String.Format(System.String,System.Object)", Justification = "Not relevant in this case")]
>
> In this case, using FormattableString.Invariant is also possible, but
> would be a little awkward, because the interpolated string is a step
> description, so it would read Invariant($"When I assign the property to
> {value})" (assuming FormattableString is imported with using static).
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/FakeItEasy/FakeItEasy/pull/713/files/3068847ef629a5d5e242d3496ea63b6cb35868d8#r63947325>
>